client/src/components/Privacy.style.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  height: 100vh;
```

Comment:
See my comment on the feature/info page! I think this makes the page bigger than it needs to be when combined with the header and footer